### DISCUSSION TITLE
ToggleButton styling

### DISCUSSION THREAD

**Discussion by Dionnelektra** [appbaseio/reactivesearch#1624] [Q&A] [answered]
Hi! I'm currently working on a project wherein I am using the ToggleButton.  I tried it in codesandbox and based on the documentation once the button is clicked by default, the text color changes into white and at the same time the button background color changes into blue.  Upon trying it in my project I have noticed that once the button is clicked only the background color of the button changes and not the text color. I have not placed any styling yet. I tried updating but still got no new result.
![Untitled](https://user-images.githubusercontent.com/22759775/109684268-11a41980-7bbb-11eb-94e6-0040599ae022.png)

**Comment by Dionnelektra** [comment, marked as answer]
I tried doing it in codesandbox 
https://codesandbox.io/s/blazing-smoke-1f8p9

**Reply by siddharthlatest** [reply]
@Dionnelektra I see it working as intended in the CodeSandbox that you shared here:

![](https://i.imgur.com/KMJjDkm.png)

Here's an example of how you can explicitly specify the styling by using the `innerClass` prop: https://codesandbox.io/s/peaceful-cray-2zgyr?file=/src/styles.css.

**Reply by Dionnelektra** [reply]
Thank you very much.
